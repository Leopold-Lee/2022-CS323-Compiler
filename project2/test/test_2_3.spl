int foo()
{
  int a = 10;
  a + 1 = 2;
  3 = 2;

  float x1 = 0.5;
  float x3[10];
  x3[x1] = x1;
  return 0;
}
