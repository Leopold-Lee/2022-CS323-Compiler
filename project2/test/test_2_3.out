Error type 6 at Line 9: left side in assignment is rvalue
Error type 6 at Line 10: left side in assignment is rvalue
Error type 12 at Line 11: indexing by non-integer
